.github/workflows/webassembly.yml

```diff
@@ -41,10 +41,9 @@ jobs:
             ~/.cargo/git
             ~/.cargo/registry
             boa_wasm/pkg
-          key: ${{ runner.os }}-yarn-build-target-${{ hashFiles('**/yarn.lock') }}
-      - uses: Borales/actions-yarn@v2.3.0
+          key: ${{ runner.os }}-npm-build-target-${{ hashFiles('**/package-lock.json') }}
+      - uses: actions/setup-node@v2
```

Comment:
They have v3 now, might be worth using that
https://github.com/actions/setup-node/releases/tag/v3.0.0